radio/src/gui/menu_model.cpp

```diff
@@ -479,6 +479,10 @@ void menuModelSelect(uint8_t event)
             eeDirty(EE_GENERAL);
           }
 
+#if defined(CPUARM)
+          checkModelIdUnique(g_eeGeneral.currModel);
```

Comment:
Does it mean you check that the receiver id is unique each time you switch from a model to another?
If this is what we want, I would prefer a check as we do for throttle / switch / ... warnings, no?